package.json

```diff
@@ -33,14 +33,18 @@
   },
   "devDependencies": {
     "browser-sync": "^2.24.5",
+    "cherio": "^1.0.0-rc.2",
     "del": "^3.0.0",
-    "gh-pages": "^1.2.0",
     "gulp": "^4.0.0",
-    "gulp-rename": "^1.2.2",
+    "gh-pages": "^1.2.0",
+    "gulp-rename": "^1.3.0",
     "gulp-replace": "^1.0.0",
-    "gulp-zip": "^4.0.0",
+    "gulp-zip": "^4.1.0",
+    "gulp-util": "^3.0.8",
```

Comment:
use of the deprecated package https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

whole gulp community is trying to remove it from projects :)